app/src/main/java/com/checkin/app/checkin/home/model/SessionOrderItemModel.kt

```diff
@@ -0,0 +1,12 @@
+package com.checkin.app.checkin.home.model
+
+import com.fasterxml.jackson.annotation.JsonProperty
+
+data class SessionOrderItemModel(
```

Comment:
You sure you couldn't have used SessionOrderedItemModel implemented in session.models package. It's already used in a similar fragment.